graphistry/Plottable.py

```diff
@@ -80,6 +80,13 @@ class Plottable(object):
     _index_to_entity : dict
 
     DGL_graph: Optional[Any]
+    
+    # embed utils
+    _relation : Optional[str]
+    _use_feat: bool
+    triplets: Optional[List]  # actually torch.Tensor too
```

Comment:
this should probably be `._triplets` ?

Reply:
(need to change the demo notebook when this updates, I know ssh one uses it, maybe others)

Reply:
I will run notebook changes